ci/terraform/oidc/sqs.tf

```diff
@@ -134,6 +134,9 @@ resource "aws_lambda_function" "email_sqs_lambda" {
       VERIFY_EMAIL_TEMPLATE_ID        = "b7dbb02f-941b-4d72-ad64-84cbe5d77c2e"
       VERIFY_PHONE_NUMBER_TEMPLATE_ID = "7dd388f1-e029-4fe7-92ff-18496dcb53e9"
       MFA_SMS_TEMPLATE_ID             = "7dd388f1-e029-4fe7-92ff-18496dcb53e9"
+      RESET_PASSWORD_TEMPLATE_ID      = "0aaf3ae8-1825-4528-af95-3093eb13fda0"
+      FRONTEND_BASE_URL               = var.frontend_base_url
```

Comment:
Doesn't this need to be `local.frontend_base_url` as we added this morning?

Reply:
good spot